src/test/java/org/openrewrite/java/testing/assertj/UseExplicitisEmptyTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.assertj;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+@SuppressWarnings("ConstantConditions")
+class UseExplicitisEmptyTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion().classpath("junit-jupiter-api", "assertj-core"))
+          .recipe(new UseExplicitIsEmpty());
+    }
+
+    @Test
+    void IsEmptyAndIsTrueBecomeIsEmpty() {
```

Comment:
```suggestion
    @Test
    @DocumentExample
    void IsEmptyAndIsTrueBecomeIsEmpty() {
```

Reply:
Then we should add the same annotation to the other test also?

Reply:
Would also need the corresponding import; this ensures we see a code snippet example in the documentation, which helps people grasp the intended change.

Reply:
> Then we should add the same annotation to the other test also?

I think a single test is enough for this recipe; not sure if we support more than one. 😅

Reply:
Ok, this should be also done.